src/main/resources/avro/biodatamethods.avdl

```diff
@@ -0,0 +1,147 @@
+@namespace("org.ga4gh.methods")
+
+protocol BiodataMethods {
+
+import idl "metadata.avdl";
+import idl "common.avdl";
+import idl "methods.avdl";
+import idl "biodata.avdl";
+
+/******************  /individuals/search  *********************/
+/**
+This request maps to the body of `POST /biosamples/search` as JSON.
+*/
+record SearchIndividualsRequest {
+  /**
+  The dataset to search within.
+  */
+  string datasetId;
```

Comment:
I had added datasetId in a previous pull request, but it was requested to leave `datasetId` out of the record itself so a sample or individual could belong to multiple datasets. https://github.com/ga4gh/schemas/pull/586